src/main/js/dashboard.js

```diff
@@ -2,41 +2,67 @@
  * User dashboard page which will present various data about accounts and transactions.
  *
  * @author Jean Dorancy
+ * @editor Xiaobing Hou
  */
 
-import React from "react";
+import React, { useState } from "react";
```

Comment:
We don't want to haves state set in the Dashboard but only in the `DashboardContainer` as React data flows from parent to children in the DOM tree. Please let's remove `useState` here.

Reply:
done